tbx/static_src/sass/components/_blockquote.scss

```diff
@@ -0,0 +1,47 @@
+@use "config" as *;
+
+.blockquote {
```

Comment:
nitpick: can we rename this class and the filename to match the block name of 'pullquote'?

Reply:
ahh good catch : ) updated in https://github.com/torchbox/torchbox.com/pull/26/commits/2b5c5fc3a553437df5a58c2f7b9b4ab6f8606f74